### ISSUE TITLE
Question about LennyFox's blocklist in Privacy Essentials

### ISSUE THREAD

**Issue by sillyjaybird** [yokoffing/filterlists#144]
I see you use Lenny Fox's Medium Mode blocklist in Privacy Essentials. How do you incorporate it given that LennyFox no longer maintains it. Just wondering if you essentially have taken over updating it for your purposes or otherwise? TIA. BTW, love your work on this project. It's an excellent resource! :)

**Comment by yokoffing** [comment]
I just incorporated it and gave credit to him in the list back in June 2023. Then I removed redundancies found in EasyList, EasyPrivacy, and Privacy Extended (AdGuard Tracking Protection) lists.

I haven't checked it since, so I didn't know he archived the project. I should probably comb through it again and see what's made its way into EasyPrivacy.

**Comment by sillyjaybird** [comment]
Thanks for your reply. One reason I ask is that I use the W3C Annual Most Used Survey List from [https://github.com/Kees1958/W3C_annual_most_used_survey_blocklist](url) in a very slim tracking prevention list in µBO. On his repo, @Lenny Fox directs users of his archived lists to @Kees1958's lists as a replacement. I try to avoid redundancies in my lists but I like Privacy Essentials.

**Comment by yokoffing** [comment]
Previously, there was a only a few false positives I removed out of the Lennyfox list. I'll use the list from Kees and see if I run into any issues.

I'll compare the two eventually. If it's better or more current, I may just remove the Lennyfox list from PE and recommend users to use the Kees list. We'll see.

**Comment by Kees1958** [comment]
@yokoffing 

When there are any fasle positives, please let me know and I will remove them. For your purpose the Mv3 versiona and addendum to Firefox and Edge are the most usefull,

Regards 

Kees

**Comment by yokoffing** [comment]
After reviewing logs, I'm not encountering sites that trigger either LennyFox or Kees list. So I will just removed LF from PE for now.

**Comment by Kees1958** [comment]
Just surf to CNN with my list only and you will see heaps of blocks. Adding a condensed list to a bulk list defeats the idea of a condensed list.

For fun compare the blocks of your largest blocklist against using my Mv3 list only.

:-)

**Comment by yokoffing** [comment]
Unique entries on CNN.com:
```
bitmovin.com^$3p
optimizely.com^$3p
srdn.com^$3p
```
Three is pretty good for an independent list (i.e. not Easylist or AdGuard).

I'm not sure about blocking Optimizely. uBlock filters unbreak has
```
! To counter `optimizely.com` in Peter Lowe's list
||optimizely.com^$badfilter
```
but that's in a first-party context.

> For fun compare the blocks of your largest blocklist against using my Mv3 list only

Any other sites you recommend?

**Comment by Kees1958** [comment]
@yokoffing 

Hal, especially for you (and for fun because I update my list yearly and am preparing for the more generous increase from 5000 to 15000 maximum dynamic rules in ManifestV3) I added a World most used list (also with RU, JP and BR networks sometimes seen on 'western world'  websites) with a whopping (at least for me, for Easylist and EasyPrivacy it is peanuts) total of 9675 third-party blockrules.


Re: Optimizely
Optimizely is an innocent A/B testing tool. Because of block count mania most adblockers started to block website optimizing and user experience optimizing tools, applying the logic that A/B testing does track users on websites. I started to include them around three years ago, because some of these A/B testing tools  also followed the user when they responded to third-part content. 

It is a mystery to me, why uBO discharges (badfilter) just one of the many A/B testing tools. I can imagine that some users complained missing out any promotions related to A/B testing. In theory one should discharge (badfilter) all A/B testing tools, to prevent missing out on promotions related to A/B testing. 

In the "block count mania" adblockers even started to block debugging tools (because they also, to some extend, track user interaction). That is why a slider like adustment (like uBO light, Ghostery and Brave have) on a per website basis is the best approach (using a minimal or standard set for trusted websites and a maximum for unknow)
